From 6366f2452865ccc7103211b8e6d8ad9e65c6a95e Mon Sep 17 00:00:00 2001
From: Konrad Dybcio <konrad.dybcio@somainline.org>
Date: Sun, 10 Oct 2021 20:16:12 +0200
Subject: [PATCH] freedreno: Enable A619

Enable A619 as found in various SKUs of the SM Lagoon SoC, such as SM6350 and
SM7225.

Signed-off-by: Konrad Dybcio <konrad.dybcio@somainline.org>
---
 src/freedreno/common/freedreno_devices.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index e13b5c6c2c1..e2d311c0d74 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -202,7 +202,7 @@ add_gpus([
 # info parameters are keyed to the sub-generation.  These templates reduce
 # the copypaste
 
-# a615, a618, a630:
+# a615, a616, a618, a619 and a630:
 a6xx_gen1 = dict(
         fibers_per_sp = 128 * 16,
         reg_size_vec4 = 96,
@@ -267,6 +267,7 @@ a6xx_gen4 = dict(
 add_gpus([
         GPUId(615),
         GPUId(618),
+        GPUId(619),
     ], A6xxGPUInfo(
         a6xx_gen1,
         num_sp_cores = 1,
-- 
2.30.1 (Apple Git-130)

